benchmarks/benchmarks/api/bench_builtin_apply_edges.py

```diff
@@ -0,0 +1,41 @@
+import time
+import dgl
+import torch
+import numpy as np
+import dgl.function as fn
+
+from .. import utils
+
+
+@utils.benchmark('time', timeout=7200)
```

Comment:
This is very large. We should have a reasonable time estimation. Does feat_size=512 on livejournal takes too long?